app/javascript/react/components/App.js

```diff
@@ -11,6 +12,7 @@ const App = (props) => {
         <Switch>
           <Route exact path="/" component={RegionsIndexPage} />
           <Route exact path="/regions/:id" component={RegionShowContainer} />
+          <Route exact path="/regions/:region_id/flavors/:id" component={FlavorShowPage} />
```

Comment:
you can make a case for the nested route `"/regions/:region_id/flavors/:id"`, but currently you arent using that region_id for anything, so as such, we really should make this `"/flavors/:id"`

Reply:
Made suggested changes